apps/site/src/app/apply/sections/Form/ResumeInformation.tsx

```diff
@@ -67,9 +66,9 @@ export default function ResumeInformation() {
 			</label>
 			<label
 				htmlFor="resume_upload"
-				className={`${styles.upload} cursor-pointer mb-3`}
+				className={`${styles.upload} cursor-pointer mb-3 p-5 rounded-xl bg-[#e1e1e1]`}
 			>
-				<Image src={uploadImage} width="100" alt="Upload resume icon" />
+				<FileText className="m-auto" width={50} height={50} />
```

Comment:
Note: the original suggestion I had was using two different icons, one for before selecting and one for after selecting a valid file. Might be cool to have a third icon for invalid (too large or wrong type) but not essential.

Reply:
Implemented the two icon suggestion in [f3ba74c](https://github.com/HackAtUCI/irvinehacks-site-2024/pull/116/commits/f3ba74c924ddace989d4630be4c1a4122f272411).

Reply:
Actually, I think we do need the tri-state: the green check icon would conflict with the invalid message shown in red below.